.coderabbit.yaml

```
@@ -1,201 +1,144 @@
   # プライバシー関連のチェック
-  privacy_check:
+  - name: "privacy_check"
     description: "プライバシー関連のコードをチェック"
+    severity: "error"
     patterns:
       - "UserDefaults"
       - "CoreData"
       - "CloudKit"
       - "HealthKit"
-    severity: "error"
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)